taskcluster/tc-cpp_tflite-tests-prod.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -xe
+
+export DEEPSPEECH_PROD_MODEL=https://github.com/lissyx/DeepSpeech/releases/download/v0.6.0/output_graph.tflite
```

Comment:
Either we land like that and we remote this once we do a 0.6.1 with the fix, or we wait for 0.6.1 before merging and we remove that right now ?

Reply:
I'd say land like that.